step-release-vis/src/app/services/data_test.ts

```diff
@@ -37,9 +37,11 @@ describe('FileService', () => {
 
   it('should return existing data', done => {
     window.localStorage.setItem('data', 'this is a test');
-    service.getData().subscribe(result => {
+    service.getLocalProtoData().subscribe(result => {
       expect(result).toEqual('this is a test');
       done();
     });
   });
+
+  // TODO(#223): add tests for new methods
```

Comment:
pls be more explict what tests are missing

Reply:
Done